integration-tests/src/test/java/tachyon/LocalTachyonClusterResource.java

```diff
@@ -104,6 +108,10 @@ public LocalTachyonClusterResource(long workerCapacityBytes, int quotaUnitBytes,
     mConfParams = confParams;
   }
 
+  public LocalTachyonClusterResource() {
+    this(DEFAULT_WORKER_CAPACITY_BYTES, DEFAULT_QUOTA_UNIT_BYTES, DEFAULT_USER_BLOCK_SIZE);
```

Comment:
By having this default constructor, a lot of cluster initialization in integration-tests can be further cleaned up (probably in a followup PR)